Source/WebKit/Platform/IPC/Connection.h

```diff
@@ -314,6 +315,12 @@ class Connection : public ThreadSafeRefCounted<Connection, WTF::DestructionThrea
     }
 
     bool sendMessage(UniqueRef<Encoder>&&, OptionSet<SendOption> sendOptions, std::optional<Thread::QOS> = std::nullopt);
+
+    struct AsyncReplyHandler {
+        CompletionHandler<void(Decoder*)> completionHandler;
+        uint64_t replyID;
```

Comment:
Ideally, new code should rely on strongly typed identifiers (using WTF::ObjectIdentifier), not raw uint64_t.

Reply:
Hopefully in the future I'll make a patch that changes the async reply id to a objectidentifier.